src/MensaWidget/MWFrameTest.class.st

```diff
@@ -48,11 +44,31 @@ MWFrameTest >> getAllCurrencies [
 	currencies := OrderedCollection new.
 	self widgetFrame allMorphsDo: [:each | 
 		(each class = MWCanteen) ifTrue: 
-			[currencies add: each submorphs first submorphs last contents]].
+			[currencies add: (self getCurrency: each)]].
 	^ currencies
 	
 ]
 
+{
+	#category : #helper,
+	#'squeak_changestamp' : 'LC 7/8/2024 22:22'
+}
+MWFrameTest >> getCanteenAmount: aCanteen [
+
+	^ aCanteen submorphs first submorphs last contents
+	
+]
+
```

Comment:
even though these are short oneliners, I'd prefer having those methods split into multiple lines and setting temp variables to make it easier to understand the context

Reply:
I just remembered there is a reason why I didn't change it and that is that I requested nils in hin ticket to implement extra variables in the morphs to access it through them because this is a case for multiple tests. It will be fixed once nils is merged but no need for me to it rn.

Reply:
perfect